build.sbt

```diff
@@ -141,23 +141,16 @@ lazy val kafka = project
 //// BENCHMARKS ////
 ////////////////////
 
-lazy val `benchmarks-vprev` = project
-  .in(file("benchmarks/vprev"))
+lazy val `benchmarks-vnext` = project
+  .in(file("benchmarks/vnext"))
+  .dependsOn(server)
   .settings(
     libraryDependencies ++= Seq(
-      "io.higherkindness" %% "mu-rpc-server" % "0.26.0"
+      "org.typelevel" %% "log4cats-core"  % V.log4cats,
+      "org.typelevel" %% "log4cats-slf4j" % V.log4cats
     )
   )
   .settings(coverageEnabled := false)
-  .settings(moduleName := "mu-benchmarks-vprev")
-  .settings(benchmarksSettings)
-  .settings(publish / skip := true)
-  .enablePlugins(JmhPlugin)
-
-lazy val `benchmarks-vnext` = project
```

Comment:
Removed this at Chris's suggestion, since it was causing binary incompatibility issues in the build, and apparently is unloved as well as causing similar issues before